pkg/cmd/server/start/master_args.go

```diff
@@ -62,6 +64,8 @@ func BindMasterArgs(args *MasterArgs, flags *pflag.FlagSet, prefix string) {
 
 	flags.Var(&args.NodeList, prefix+"nodes", "The hostnames of each node. This currently must be specified up front. Comma delimited list")
 	flags.Var(&args.CORSAllowedOrigins, prefix+"cors-allowed-origins", "List of allowed origins for CORS, comma separated.  An allowed origin can be a regular expression to support subdomain matching.  CORS is enabled for localhost, 127.0.0.1, and the asset server by default.")
+
+	flags.StringVar(&args.SchedulerConfigFile, prefix+"scheduler-config-file", args.SchedulerConfigFile, "File with scheduler policy configuration")
```

Comment:
My goal is to avoid adding any more flags unless they are directly needed for driving test-end-to-end or for common development startup scenarios.

The sooner we can get access to the serializable config pre-startup in integration tests, the stronger our case for avoiding startup flags will be.